src/scripts/core/models/timer.ts

```diff
@@ -25,10 +25,8 @@ export class Timer extends Container {
     }
 
     private formatTime(): void {
-        let minutes = "0";
-        let seconds = "0";
-        this._currentTime >= 60 ? minutes = Math.floor(this._currentTime / 60).toString() : minutes = '0';
-        this._currentTime % 60 === 0 ? seconds = "00" : seconds = (this._currentTime % 60).toString().padStart(2, "0");
+        const minutes: String = this._currentTime >= 60 ? Math.floor(this._currentTime / 60).toString() : '0';
+        const seconds = (this._currentTime % 60).toString().padStart(2, "0");
```

Comment:
I'm going to be that guy.. If you are typing one you should be typing the other. It is, however, okay to not type either of them here as the type will be inferred and you don't have to worry about a different type being assigned as it is const.

Reply:
Ah fair one. I put in the type to resolve an error and missed adding it on to seconds. I'll change it.